docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
-          }
-        ]
-      }
-      ```
-
-  - Example 2: 
-
-    - ```python
-      message = 'Please call me'
-      entity_name = 'phone_number'
-      structured_value = None
-      fallback_value = '9049961794'
-      bot_message = None
-      ```
+The Time detector module has the capability to detect time from text in multiple languages. It can detect time in 12/24 hr format and also detect text with time difference specified. for example - wake me up `after 10 mins`. 
 
-    - *Python:* 
+Currently time detection support has been provided in different languages - `English`,  `Hindi`, `Marathi`,  `Bengali`,  `Gujrati`, `Tamil`. It also supports latin script of given languages.
 
-      ```python
-      from ner_v1.chatbot.entity_detection import get_phone_number
-      output = get_phone_number(message=message, entity_name=entity_name, structured_value=structured_value, fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
-
-    - *CURL command:*
-
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
-
-      ​
-
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/phone_number/?message=Please%20call%20me&entity_name=phone_number&structured_value=&fallback_value=9049961794&bot_message='
-      ```
-
-    - *CURL Output:*
-
-      ```json
-      {
-        "data": [
-          {
-            "detection": "fallback_value",
-            "original_text": "9049961794",
-            "entity_value": {
-              "value": "9049961794"
-            }
-          }
-        ]
-      }
-      ```
-
-### email
-
-- This functionality calls the EmailDetector class to detect email ids.
-
-- Example:
-
-  - Example 1: 
-
-    - ```python
-      message = 'my email id is apurv.nagvenkar@gmail.com'
-      entity_name = 'email'
-      structured_value = None
-      fallback_value = None
-      bot_message = None
-      ```
-
-    - *Python:*
-
-      ```python
-      from ner_v1.chatbot.entity_detection import get_email
-      output = get_email(message=message, entity_name=entity_name, structured_value=structured_value,fallback_value=fallback_value, bot_message=bot_message)
-      print output
-      ```
-
-    - *CURL command:*
-
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
-
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/email/?message=my%20email%20id%20is%20apurv.nagvenkar%40gmail.com&entity_name=email&structured_value=&fallback_value=&bot_message='
-      ```
-
-    - *CURL Output:*
-
-      ```json
-      {
-        "data": [
-          {
+> To add support for new languages or to add custom patterns in particular language please go through the Time Detector readme [here](https://github.com/hellohaptik/chatbot_ner/blob/develop/ner_v2/detectors/temporal/time/README.md)
+
+*Note - This module has been updated to v2 version of chatbot_ner.*
+
+**API Example**:
+
+- ***Example 1: Detect time[English] from text having 24 hrs, 12 hrs and time difference time format***
```

Comment:
Can be reframed
Detect time from English text which contains time mentions in 24 hours format, 12 hours format and relative time format